src/utilities/multipleChoice.js

```diff
@@ -0,0 +1,19 @@
+import { singleAnswerQuestion } from './singleAnswer';
+
+export class multiChoiceQuestion extends singleAnswerQuestion {
```

Comment:
I'm not the biggest fan of `MultiChoiceQuestion` extending the `SingleAnswerQuestion`. The multiple choices question is not a type of a single answer question. It looks a bit like Labrador extending the Cat class.

I suggest you create a generic `Question` class.

Reply:
Also, "single answer" sounds a little too similar to "single choice"